tests/ipex/test_modeling.py

```diff
@@ -297,6 +301,10 @@ def test_ipex_patching_beam_search(self, test_name, model_arch, use_cache):
                 self.assertIsInstance(outputs, torch.Tensor)
                 self.assertTrue(torch.equal(outputs, transformers_outputs))
 
+    @unittest.skipIf(
+        is_ipex_version(">=", "2.3.0"),
+        reason="CPU IPEXModel only supports with past_key_values for ipex version >= 2.3.0",
+    )
```

Comment:
this model is already exported so should work no ?
```suggestion
```
but makes sense for patched models (`_is_ipex_exported=True`) to explicitly  state that `use_cache` cannot be set to False